packages/cli/tests/unit/agentInstall/javaBuildToolInstaller.spec.ts

```diff
@@ -16,20 +16,21 @@ describe('JavaBuildToolInstaller', () => {
   });
 
   describe('when IntelliJ is installed', () => {
-    const intellijHome = '/IntelliJ';
+    const intellijHome = (process.platform === 'win32' ? '\\' : '/') + 'IntelliJ';
```

Comment:
You could also use `path.sep` or, even clearer, `path.join('/', 'IntelliJ')` (I checked, it does the right thing on windows.)

Reply:
Yeah, that's cleaner. I changed it.